app/views/users/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Editing User</h1>
+
+<%= form_for @user do |f| %>
```

Comment:
You can see that this is the same code that you have in `new.html.erb`, which is an indication that it is a good case for a partial. You can replace your current `_form.html.erb` template with the content of this form and then you can render that form in both templates.

[OPTIONAL]